android/app-project-newm/src/main/java/io/projectnewm/NewmAppComposable.kt

```diff
@@ -80,52 +95,60 @@ private fun DebugMenuButton() {
-                onClick = { onNavigationSelected(Screen.HomeRoot) }
-            )
-            HomeBottomNavigationItem(
-                icon = R.drawable.ic_tribe,
-                selected = currentRootScreen == Screen.TribeRoot,
-                label = stringResource(R.string.tribe),
-                onClick = { onNavigationSelected(Screen.TribeRoot) }
-            )
-            HomeBottomNavigationItem(
-                icon = R.drawable.ic_stars,
-                selected = currentRootScreen == Screen.StarsRoot,
-                label = stringResource(R.string.stars),
-                onClick = { onNavigationSelected(Screen.StarsRoot) }
-            )
-            HomeBottomNavigationItem(
-                icon = R.drawable.ic_wallet,
-                selected = currentRootScreen == Screen.WalletRoot,
-                label = stringResource(R.string.wallet),
-                onClick = { onNavigationSelected(Screen.WalletRoot) }
-            )
+    AnimatedVisibility(
+        visible = isVisible,
+        enter = slideInVertically(initialOffsetY = { it }),
+        exit = slideOutVertically(targetOffsetY = { it }),
+    ) {
+        Column(Modifier.height(76.dp)) {
+            NewmRainbowDivider()
+            BottomNavigation(
+                backgroundColor = Color.Black,
+                contentColor = Color.White,
+                modifier = Modifier
+                    .fillMaxHeight()
+                    .testTag(TAG_BOTTOM_NAVIGATION)
+            ) {
+                HomeBottomNavigationItem(
+                    icon = R.drawable.ic_home,
+                    selected = currentRootScreen == Screen.HomeRoot,
+                    label = stringResource(R.string.home),
+                    onClick = { onNavigationSelected(Screen.HomeRoot) }
+                )
+                HomeBottomNavigationItem(
+                    icon = R.drawable.ic_tribe,
+                    selected = currentRootScreen == Screen.TribeRoot,
+                    label = stringResource(R.string.tribe),
+                    onClick = { onNavigationSelected(Screen.TribeRoot) }
+                )
+                HomeBottomNavigationItem(
+                    icon = R.drawable.ic_stars,
+                    selected = currentRootScreen == Screen.StarsRoot,
+                    label = stringResource(R.string.stars),
+                    onClick = { onNavigationSelected(Screen.StarsRoot) }
+                )
+                HomeBottomNavigationItem(
+                    icon = R.drawable.ic_wallet,
+                    selected = currentRootScreen == Screen.WalletRoot,
+                    label = stringResource(R.string.wallet),
+                    onClick = { onNavigationSelected(Screen.WalletRoot) }
+                )
+            }
         }
     }
 }
 
 @Preview(showBackground = true)
 @Composable
 fun BottomNavigationBarPreview() {
-    NewmBottomNavigation(Screen.HomeRoot) {}
+    NewmBottomNavigation(Screen.HomeRoot, true) {}
```

Comment:
Is there a different approach we can take? When we had the detail page example it took the entire screen, can do something similar for the now playing?

I don't see the case where we would need to show the bottom navigation if we are not in the main screens. And we will need to pass in false for any subsequent screens, which there will probably be more than the numbers of screens we have on our bottom nav

Reply:
This is the best way I could figure out for now with compose.  We can address it later once we get more screens but for now  it's straightforward enough to just add the  route to the `routesWithoutBottomNavBar` list and the rest will happen automatically

Reply:
If you have another suggestion I'm game to try it but otherwise let's keep this as-is for now since it works great for the demo